src/main/java/teammates/logic/backdoor/BackDoorLogic.java

```diff
@@ -430,6 +438,9 @@ public void removeDataBundle(DataBundle dataBundle) {
             }
         }
         accountsDb.deleteAccounts(dataBundle.accounts.values());
+
+        // Delete all admin emails
+        adminEmailsDb.deleteEntities(adminEmailsDb.getAllAdminEmails());
```

Comment:
Sorry! I forgot that the tests are also ran on the server. Should the commit be reverted?